### DISCUSSION TITLE
Şişe ve kapaklar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#243] [Zeka Sorusu] [answered]
4 Şişe kapağı getirene bir meyve suyu verilen bir kampanyada, elinde 100 kapak olan bir kişi bu kampanyadan ne kadar meyve suyu içebilir?

**Comment by ElifNurSoyler** [comment, marked as answer]
100÷4=25 (meyve suyu) + 25 şişe kapağı
25÷4=6 (meyve suyu) + 6 şişe kapağı + 1 şişe kapağı
7÷4=1 (meyve suyu) +  1 şişe kapağı + 3 şişe kapağı
4÷4=1 (meyve suyu) + 1 şişe kapağı
Toplam = 33 tane meyve suyu içmiştir.
